packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/useModeledDataStreams/useModeledDataStreams.ts

```diff
@@ -29,12 +30,12 @@ export function useModeledDataStreams({ assetIds, client }: UseModeledDataStream
 
 function createQueryFn(client: IoTSiteWiseClient) {
   return async function ({
-    queryKey: [{ assetId }],
+    queryKey: [assetProperties],
     signal,
   }: QueryFunctionContext<ReturnType<ModeledDataStreamCacheKeyFactory['create']>>) {
-    invariant(assetId, 'Expected assetId to be defined as required by the enabled flag.');
+    invariant(assetProperties, 'Expected assetProp to be defined as required by the enabled flag.');
 
-    const request = new ListModeledDataStreamsRequest({ assetId, client, signal });
+    const request = new ListModeledDataStreamsRequest({ assetProp: assetProperties.assetProp, client, signal });
```

Comment:
By destructuring the queryKey in the function parameters, such as `queryKey: [{ assetProp }]`, you can change the request to `const request = new ListModeledDataStreamRequest({ assetProp, client, signal });` and make things a little cleaner. :)